docs/changelog.md

```diff
@@ -4,6 +4,18 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.5.2
+
+- FOREACH clause using outer variable scope to allow nesting. [#725](https://github.com/memgraph/memgraph/pull/725)
```

Comment:
What's the point here? Is the whole foreach clause new, or is just new that it now uses outer variable scope? what can i use this for? why is this helpful?

Reply:
No, the FOREACH isn't new. The PR has issues linked that describe the problem. In short, we couldn't see variables (symbols) inside foreach that were defined outside of it, so we couldn't for instance nest a couple of foreach-es and use their variables. I will not go deeper that that because it gets technical and we had a few discussions about this. You can see more in the last comments of the PR but the main functionality improvement is from Joerg's issue.

Reply:
Can we just write: 

Nesting several FOREACH clauses now work as expected.  

That's the point right? If I nest a couple of FOREACH the calculation will be correct?